src/components/reactions.vue

```diff
@@ -102,5 +102,22 @@ export default defineComponent({
 })
 </script>
 <style scoped>
-
+.plusButton{
+  -webkit-font-smoothing: antialiased;
+  margin: 4px;
+  padding: 7px 1px;
+  display: inline-flex;
+  position: relative;
+  align-items: center;
+  height: 32px;
+  background: var(--background);
+  color: var(--color);
+  font-family: var(--ion-font-family, inherit);
+  font-size: 14px;
```

Comment:
Are all of them really necessary? And are they responsive? `ionic` already provides a lot of helpers and sizes, using `px` here almost certainly means there will be issues with other sizes.

Reply:
Yes, define properties about the class which I have used. This is not used as root, so using `px` not effect the sizes on others too

Reply:
I don't mean other elements, mainly other font-sizes, I mean if were to change the theme of ionic. I don't see why we'd need to set color, font-family, font-size here at all, these should all be inherited from button (same of cursor-pointer). Please go through these add remove everything that isn't strictly necessary to get the alignment fixed - we should only change elements minimally on this lowest level to not mess with broader ionic theming facilities.

Reply:
I have removed the other elements